tests/transport/spdp/run_test.pl

```diff
@@ -0,0 +1,19 @@
+eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
```

Comment:
Once the test validation code is ready, make sure to add this `run_test.pl` to the overall list in `dcps_tests.lst`.